community/webkit2gtk/build

```diff
@@ -1,5 +1,15 @@
 #!/bin/sh -e
 
+# Enable streaming related options if gstreamer
+# plugins are installed.
+kiss l gstreamer gst-plugins >/dev/null 2>&1 &&
+    streaming=ON
+
+# Enable the use of openjpeg if the package is
+# installed.
+kiss l openjpeg2 >/dev/null 2>&1 &&
+    jpeg=ON
```

Comment:
It's probably fine to see output from these. The terminal will endlessly scroll afterwards anyway. :P

Reply:
You can probably then fit each one on a single line.

Reply:
Yeah you're right :D